Refactor expect tests on test_higher_order_ops.py.

ysiraichi opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #110417
* #107618
* #109433
* __->__ #110290

This PR inlines the expecteds strings onto the `assertExpectedInline` calls, so that, when
change is needed, we may do that by using the `expectedtest` machinery: setting the
environment variable `EXPECTTEST_ACCEPT=1`.

cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @Xia-Weiwen @wenzhe-nrv @jiayisunx @chenyang78 @aakhundov @kadeng

ysiraichi committed:
Refactor expect tests on test_higher_order_ops.py.

This PR inlines the expecteds strings onto the `assertExpectedInline` calls, so that, when
change is needed, we may do that by using the `expectedtest` machinery: setting the
environment variable `EXPECTTEST_ACCEPT=1`.

[ghstack-poisoned]

ysiraichi committed:
Fix lint issues. on "Refactor expect tests on test_higher_order_ops.py."


This PR inlines the expecteds strings onto the `assertExpectedInline` calls, so that, when
change is needed, we may do that by using the `expectedtest` machinery: setting the
environment variable `EXPECTTEST_ACCEPT=1`.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng Xia-Weiwen wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]

kit1980 commented:
@pytorchbot revert -m "Broke multiple tests and also lint https://github.com/pytorch/pytorch/actions/runs/6384854768/job/17329068768" -c ignoredsignal

kit1980 commented:
> @pytorchbot revert -m "Broke multiple tests and also lint https://github.com/pytorch/pytorch/actions/runs/6384854768/job/17329068768" -c ignoredsignal

Actually this one may be OK and the next one in the stack is the problem, but because it was landed as a stack, the CI was run only on the last PR.

ysiraichi committed:
Rebased. on "Refactor expect tests on test_higher_order_ops.py."


This PR inlines the expecteds strings onto the `assertExpectedInline` calls, so that, when
change is needed, we may do that by using the `expectedtest` machinery: setting the
environment variable `EXPECTTEST_ACCEPT=1`.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng Xia-Weiwen wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]
